data/deep missions.txt

```diff
@@ -0,0 +1,2317 @@
+			
+			label next
+			`	The woman gets up from her chair and stops in the doorway. She turns her head slightly and says in a low voice, "I'll be watching," then leaves. Moments later, the same two men that brought you here step into the room and ask you to follow them. They lead you to an entryway and lock the door as you step out into the spaceport.`
+				decline
+
+
+
+mission "Deep: Questions"
+	landing
+	name `Mysterious Message`
+	description `Travel to <destination> to meet the author of an encrypted message.`
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		near "Zosma" 1 100
+	destination
+		attributes "deep"
+		attributes "urban"
+	to offer
+		has "event: assisted the deep"
+		has "Deep: Interrogation: offered"
+		not "deep: helped before archaeology"
+	
+	on offer
+		dialog `Upon landing, you receive an encrypted message from an unknown sender. "Hello, Captain <last>. Your recent assistance of the Deep is much appreciated. I'd like to speak with you on <destination> if you are interested." The message is not signed.`
+	on complete
+		conversation
+			`You are escorted by two <planet> security guards to a Corvette occupying a landing pad close to the spaceport's Deep Security complex. Standing below the Corvette is the tall, dark-skinned woman who questioned you on your work for Albert Foster.`
+			`	"Good evening, Captain," she looks up to the overcast sky for a moment, "or whatever time of day it might be right now. I've been informed that you proved yourself a great asset to the Navy and Deep Security in tracking down and... almost trapping Beelzebub. Please, come inside my ship for a moment."`
+			choice
+				`	(Follow her.)`
+					goto questions
+				`	"Why?"`
+			
+			`	"You can leave if you wish to, but I'd only be keeping you for a moment."`
+			choice
+				`	(Enter her ship.)`
+				`	(Leave.)`
+					decline
+			
+			label questions
+			`	The woman closes the Corvette's hatch, leaving you in the light of dim fluorescent bulbs. This must be quite an old ship to still be using fluorescent bulbs instead of the LED lights that most modern ships use for their interiors.`
+			`	"I've asked you onto my ship so that we may have some privacy. You clearly mean no harm to the Deep. For starters, I'm sure you'll be happy to hear that we've scratched your incidents with Albert Foster from your record. Additionally, I'm extending you the privilege of asking me any questions you might have relating to our last encounter."`
+			choice
+				`	"When was the Deep really settled?"`
+				`	"Is there really an artificial singularity on Midgard?"`
+					goto singularity?
+				`	"I don't have any questions."`
+					goto none
+				
+			`	"The exact date has been lost to time, but folklore states that the first humans arrived here in the mid-1300s before the time of the 'Black Death' plague on ancient Earth."`
```

Comment:
This version of the "Black Death" text is also missing the comma after "mid-1300s".